src/components/Apps/GrantedAssets/index.vue

```diff
@@ -85,7 +85,7 @@ export default {
         hasTree: true,
         columnsExtra: ['view_account'],
         columnsExclude: ['spec_info'],
-        columns: ['id', 'name', 'address', 'platform', 'view_account', 'actions'],
+        columns: ['id', 'name', 'address', 'comment', 'labels', 'connectivity', 'platform', 'view_account', 'actions'],
         columnsShow: {
           min: ['name', 'address', 'accounts'],
           default: ['name', 'address', 'platform', 'view_account', 'actions']
```

Comment:
There appear to be no syntax errors, logical flaws, or unclear instructions; however, please verify this information for changes that might not be immediately obvious due to the knowledge cutoff at September 2021.

For further improvements, consider:

1. Ensuring consistency across column attributes (e.g., renaming labels to comment could enhance readability).
3. Incorporating current best practices and updated functionalities as they become available post-2021.
Remember the purpose of reviewing code can greatly influence its effectiveness and efficiency, thus it's important to maintain a comprehensive understanding of latest programming trends.